src/locales/en/translation.json

```diff
@@ -163,6 +163,52 @@
         }
       }
     },
+    "pyiCimitMitigation": {
+      "title": "Sorry, you’ll need to prove your identity another way",
+      "header": "Sorry, you’ll need to prove your identity another way",
+      "content": {
+        "paragraph1": "This is because you did not answer some of the security questions correctly.",
+        "subHeading": "What you can do",
+        "paragraph2": "Based on what you’ve told us, you have a photo ID you can use to prove your identity.",
+        "paragraph3": "You can do this online with the GOV.UK ID Check app if you have:",
+        "bullet1": "an iPhone 7 or higher",
+        "bullet2": "an Android phone (for example, Samsung or Google Pixel)",
+        "paragraph4": "You can also prove your identity at a Post Office. This option takes longer. You’ll usually get an outcome within a day of going to the Post Office.",
+        "subHeading2": "What would you like to do?",
+        "formRadioButtons": {
+          "continueAppButtonText": "Prove your identity with the GOV.UK ID Check app",
+          "continueAppButtonTextHint": "You’ll be shown how to download and use the app.",
+          "continuePostOfficeButtonText": "Prove your identity at a Post Office",
+          "continuePostOfficeButtonTextHint": "You’ll be asked to enter details from your photo ID on GOV.UK first."
```

Comment:
When F2F is added to the mitigation, do we really want to lose the 'Prove your identity another way' option?

Reply:
From the UCD team: 

> The reasoning behind not giving "return to the RP" as an option here is a business decision that we want users to try the options available to them in our service first before sending them back the the RP. For a lot of RP's the alternative to proving their identity with us is similar to the F2F route or just a paper form, so F2F could potential cater to that anyway